src/LinuxTracing/UprobesUnwindingVisitorTest.cpp

```diff
@@ -68,6 +68,10 @@ class MockLibunwindstackUnwinder : public LibunwindstackUnwinder {
               (pid_t, unwindstack::Maps*, (const std::array<uint64_t, PERF_REG_X86_64_MAX>&),
                const void*, uint64_t, bool, size_t),
               (override));
+  MOCK_METHOD(LibunwindstackResult, Unwind,
+              (pid_t, unwindstack::Maps*, (const std::array<uint64_t, PERF_REG_X86_64_MAX>&),
+               const void*, uint64_t, const void*, uint64_t, uint64_t, bool, size_t),
+              (override));
```

Comment:
Is this from a different change? No method is being added to `LibunwindstackUnwinder`.

Reply:
Probably why the builds fail.

Reply:
Yep, missed to remove that part from the PR. Sorry for the noice.